src/models/externals/IExternalRoutePathNode.ts

```diff
@@ -0,0 +1,15 @@
+import IExternalRoutePath from './IExternalRoutePath';
+
+export default interface IExternalroutePathNode {
```

Comment:
One more very confusing naming. Could this just be a node? How is this different from a node?

Reply:
Outdated, we now only have IExternalNode -> resolving